content/content-strategy/manage-content-requests/plan-content/content-planning-workshop/toc.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: nav/sections
+sections:
+  - Why run a workshop
```

Comment:
Broken link: needs the `?` after it so that it matches what's in `content/content-strategy/manage-content-requests/plan-content/content-planning-workshop/why-head.md`

Reply:
Trisha: done